alphastats/gui/gui.py

```diff
@@ -0,0 +1,7 @@
+import os
+
+
+def run():
+    file_path = os.path.realpath(__file__)
+    os.chdir(os.path.dirname(file_path))
+    os.system("streamlit run AlphaStats.py")
```

Comment:
This could cause issues because it is not clear which streamlit installation is used.